Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -883,58 +882,58 @@ class ScrollView extends React.Component<Props, State> {
   }
 
   // [TODO(macOS ISS#2323203)
-  _handleKeyDown = (e: KeyboardEvent) => {
-    if (this.props.onKeyDown) {
-      this.props.onKeyDown(e);
+  _handleKeyDown = (e: ScrollEvent) => {
+    if (this.props.onScrollKeyDown) {
+      this.props.onScrollKeyDown(e);
     } else {
       const event = e.nativeEvent;
```

Comment:
Probably beyond scope (adds more unnecessary diff?) to rename this to nativeEvent and "e" to "event"?

Reply:
no problem